### DISCUSSION TITLE
Understanding how DIA-NN combines peptides to proteins

### DISCUSSION THREAD

**Discussion by igalriklin** [vdemichev/DiaNN#906] [General]
Dear Vadim,

I was trying to understand how the intensities values of the proteins in the protein table are derived from the peptide intensities values in the peptides table.
I found in the article that describes DIA-NN that:
"A ‘top 3’ method is eventually used for protein quantification: intensities of protein groups are calculated as sums of the intensities of the three most abundant precursors identified with a q-value lower than a given threshold in a particular acquisition."

To see if I understood correctly, I have found in my peptides file a protein group that has a small amount of peptides precursors:

 | Protein.Group | Protein.Ids | Protein.Names | Genes | First.Protein.Description | Proteotypic | Stripped.Sequence | Modified.Sequence | Precursor.Charge | Precursor.Id | C31.raw | C45.raw | 
 | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | 
 | A0A075B6Q5 | P0DP02;A0A075B6Q5 | HV364_HUMAN | IGHV3-64 | Immunoglobulin heavy variable 3-64 | 0 | LSCAASGFTFSSYAMHWVR | LSC(UniMod:4)AASGFTFSSYAMHWVR | 3 | LSC(UniMod:4)AASGFTFSSYAMHWVR3 | 238168 | 155903 | 
 | A0A075B6Q5 | A0A075B6Q5 | HV364_HUMAN | IGHV3-64 | Immunoglobulin heavy variable 3-64 | 1 | NTLYLQMGSLR | NTLYLQMGSLR | 2 | NTLYLQMGSLR2 | 338473 | 754450|

and here is the values of this proteine in the proteine table:

|Protein.Group|Protein.Ids|Protein.Names|Genes|First.Protein.Description|C31.raw|C45.raw|
|---|---|---|---|---|---|---|
|A0A075B6Q5|A0A075B6Q5|HV364_HUMAN|IGHV3-64|Immunoglobulin heavy variable 3-64|459781|555397|

The value of the protein for C31.raw is 459781. However it is not the sum of the values of the peptides of C31.raw, which are 238168, 338473.
I looked at other proteins too, and I saw the same problem there.

Could you help me figure out what am I missing? Is the current version of DIA-NN different from what is described in the article regarding the intensities of the proteins?

Thanks,
Igal.

**Comment by vdemichev** [comment]
Hi Igal,

DIA-NN now uses a MaxLFQ-like algorithm, not Top 3. 
So it's not possible to figure it out 'manually', the quantities are a solution of a certain matrix equiation.

Best,
Vadim

**Reply by igalriklin** [reply]
Thanks for your help!
Now I understand how to calculate the ratio in the quantities between the different samples of each protein. I tried it manualy for an example of a protein with 2 peptides and found the correct ratio between the samples.
However the total sum of the peptides is still not the quantity of the proteins. Is it because there is a normalization step on the quantities of the peptides prior to their combination into proteins?

Thanks,
Igal.

**Reply by vdemichev** [reply]
Hi Igal,

'However the total sum of the peptides is still not the quantity of the proteins' - what do you mean? MaxLFQ does not sum peptides.

Best,
Vadim

**Reply by igalriklin** [reply]
Hello Vadim,
I meant that in MaxLFQ article it is written:
"We perform a least-squares analysis to reconstruct the ... individual protein ratios"
and right after that:
"Then we rescale the whole profile to the cumulative intensity across samples, thereby preserving the total summed intensity for a protein over all samples"
I was trying to understand what does it mean "preserving the total summed intensity for a protein over all samples". In others words, after we know the ratio of the quantities of the samples, how do we find the absolte scale? Is there some kind of normalization?

Thanks,
Igal.

**Reply by vdemichev** [reply]
Yes, this rescaling does not apply to DIA-NN. DIA-NN does precursor-level normalisation and then figures out protein intensities. Those are for relative quant only, there are no guarantees provided for absolute quant, i.e. cannot compare different protein species.
